### PR TITLE
Samsung.BatteryManager version 2.2.1.3

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111979&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
image

File: manifests/s/Samsung/BatteryManager/2.2.1.3/Samsung.BatteryManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: Samsung.BatteryManager
+PackageVersion: 2.2.1.3
+Installers:
+- Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: setup.exe
+  InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=1830317&CDCttType=SW&ModelType=C&ModelName=NP-Q1U/P01/SEA&VPath=SW/200801/20080122135939796_Samsung_Battery_Manager_2.2.1.3.zip
```
Comment (Trenly) [inline manifests/s/Samsung/BatteryManager/2.2.1.3/Samsung.BatteryManager.installer.yaml:12]:
```suggestion
  InstallerUrl: https://downloadcenter.samsung.com/content/SW/200801/20080122135939796_Samsung_Battery_Manager_2.2.1.3.zip
```

File: manifests/s/Samsung/BatteryManager/2.2.1.3/Samsung.BatteryManager.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.BatteryManager
+PackageVersion: 2.2.1.3
```
Comment (Trenly) [inline manifests/s/Samsung/BatteryManager/2.2.1.3/Samsung.BatteryManager.installer.yaml:5]:
```suggestion
PackageVersion: 1.00
```

File: manifests/s/Samsung/BatteryManager/2.2.1.3/Samsung.BatteryManager.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.BatteryManager
+PackageVersion: 2.2.1.3
```
Comment (Trenly) [inline manifests/s/Samsung/BatteryManager/2.2.1.3/Samsung.BatteryManager.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.00
```

File: manifests/s/Samsung/BatteryManager/2.2.1.3/Samsung.BatteryManager.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Samsung.BatteryManager
+PackageVersion: 2.2.1.3
```
Comment (Trenly) [inline manifests/s/Samsung/BatteryManager/2.2.1.3/Samsung.BatteryManager.yaml:5]:
```suggestion
PackageVersion: 1.00
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by stephengillie** [comment]
Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\Samsung\BatteryManager\2.2.1.3', expected 'manifests\s\Samsung\BatteryManager\1.00'

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Pull request closed without merging**